operators/argo-controller/src/charm.py

```diff
@@ -87,14 +92,8 @@ def main(self, event):
             },
         }
 
-        crd_paths = [
-            Path(f"files/{crd_file}")
-            for crd_file in listdir("files")
-            if Path(f"files/{crd_file}").is_file()
-        ]
-
-        crds = [yaml.safe_load(crd_path.read_text()) for crd_path in crd_paths]
-
+        crd_root = "src/crds"
```

Comment:
I understand you are doing this because of the recent changes in charmcraft which don't include non default files. 

I don't entirely like having crds in the `src` directory, according to [this](https://discourse.charmhub.io/t/charm-packing-will-not-include-all-files-by-default-anymore/4924?u=dominik.f) we can include the `files` directory in build by adding the `parts` section in the `charmcraft.yaml`.

Reply:
Yeah I had it pulling from `files` using `charmcraft.yaml` at first, but that had strange [consequences] during build.  Looks like they're now fixed but haven't tested.  I lean slightly to preferring the crds be kept outside `/src` too if it actually works, but there was some discussion [here](https://github.com/canonical/argo-operators/pull/8/files/e975f3219eb75c78856a414700c6a073377fbf8b#diff-4b15a4373768a97d7b037278a2ab95093d43880f51d1ee55e7f8f6ca15f9c489).

Reply:
Moved them back